python/ray/rllib/ppo/ppo.py

```diff
@@ -104,14 +103,13 @@ def _init(self):
             self.file_writer = None
         self.saver = tf.train.Saver(max_to_keep=None)
 
-    def train(self):
+    def _train(self):
         agents = self.agents
         config = self.config
         model = self.model
-        j = self.j
-        self.j += 1
+        j = self.iteration
```

Comment:
want to get rid of "j" and replace with self.iteration? It was names that since it used to be a loop counter, but now "iteration" makes a whole lot more sense...

Reply:
Done